.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Thanks for your interest in The Odin Project. As a courtesy to our maintainers please do a search in our issues to make sure this is not a duplicate of an existing issue. In order to get issues addressed in a reasonable amount of time, we request that you include a baseline of information about the issue you're experiencing and how to reproduce it. Please provide the following:
+
+### Describe the nature of the problem you're experiencing:
+
+...your issue
+
+### Describe what you think *should* happen:
+
+...expected behavior
+
+### Describe what actually *is* happening:
+
+...problem behavior
+
+### If the issue related to the rendering or content of the website, please include a screenshot of the problem area.
+
+...screenshot if needed
```

Comment:
Should we make a mention here about how they can attach images by just pasting them into github?